idl/internal/lex.rl

```diff
@@ -44,6 +46,7 @@ type lexer struct {
 func newLexer(data []byte) *lexer {
     lex := &lexer{
         line: 1,
+        tabWidth: 8,
```

Comment:
8 feels like the right value because it's (a) the definition of a tab and (b) it's what most terminals will render.

I expect many people use 4-column tabs in their editors, though.